docs/modules/ROOT/pages/how-to-guides/proc_delete_application.adoc

```diff
@@ -18,5 +19,23 @@ WARNING: If you delete an application permanently, you delete all the components
 +
 NOTE: You can also delete an application by selecting *Actions > Delete Application*.
 
-== Additional Resources
-For information about deleting an application from CLI, refer to xref:how-to-guides/delete_application_cli.adoc[Deleting an application from CLI].
\ No newline at end of file
+
+== Deleting an application using CLI
+
+.*Prerequisites*
+
+* You successfully signed in to {ProductName}.
```

Comment:
You have successfully signed into {ProductName}. 

This way, both prereqs have consistent sentence structure. Also, minor error with `in to` should be `into`